lib/ui/text.dart

```diff
@@ -374,25 +365,24 @@ class TextStyle {
 // The encoded array buffer has 4 elements.
 //
 //  - Element 0: A bit field where the ith bit indicates wheter the ith element
-//    has a non-null value. Bit 6 indicates whether |fontFamily| is non-null.
-//    Bit 7 indicates whether |fontSize| is non-null. Bit 8 indicates whether
-//    |lineHeight| is non-null. Bit 0 is unused.
```

Comment:
Shouldn't we keep the documentation about the bits beyond the last element of the list?